qiskit_dynamics/models/hamiltonian_model.py

```diff
@@ -34,14 +38,14 @@ class HamiltonianModel(GeneratorModel):
     .. math::
         H(t) = H_d + \sum_j s_j(t) H_j,
 
-    where :math:`H_j` are Hermitian operators, :math:`H_d` is the drift component,
+    where :math:`H_j` are Hermitian operators, :math:`H_d` is the static component,
     and the :math:`s_j(t)` are either :class:`~qiskit_dynamics.signals.Signal` objects or
     are numerical constants. Constructing a :class:`~qiskit_dynamics.models.HamiltonianModel`
     requires specifying the above decomposition, e.g.:
 
     .. code-block:: python
 
-        hamiltonian = HamiltonianModel(operators, signals, drift)
+        hamiltonian = HamiltonianModel(operators, signals, static_operator)
```

Comment:
The signature of HamiltonianModel below is `(static_operator, operators, signals)` so the above would give an error. You either need to change the order or use kwargs:
```suggestion
        hamiltonian = HamiltonianModel(static_operator=static_operator, operators=operators, signals=signals)
```